### ISSUE TITLE
No longer able to access database file

### ISSUE THREAD

**Issue by wpeace3** [fguillot/kanboard#1122]
Working in Kanboard recently updating a task and found I was unable to navigate back to the board. 

After a couple of attempts with various results none of which allowed me to navigate out of the task I exited the browser window. I started a new browser session and was greeted with a blank screen. Multiple attempts to access the site elicit the same results. I've upgraded the site to the latest version of kanboard (v1.0.17) with no change in the behavior.

I'd assumed the problem was with the site, but I also checked the database file. It is still readable BUT, moving the database file to my test server and replacing the database file there which is working I'm getting the same result. A blank screen when accessing the site.

Has anyone seen this problem or have an idea how to correct?

**Comment by fguillot** [comment]
- Do you have any php error visible in your log files?
- Can you open the database with the `sqlite3` command and read the data?
- Are you using a low cost shared-hosting provider? if yes, are they using NFS to store your files?

**Comment by wpeace3** [comment]
*         Where are the log files?

*         I dont have shell access however I did copy the file to my PC and
I can read the data using a sqlite db manager. The data looks good and I was
able to get a dump just in case I had to recreate it.

*         No, the database was move to my local server and I experienced the
same problem. An older version of the database was on my old server and I
have no problem starting the app.
